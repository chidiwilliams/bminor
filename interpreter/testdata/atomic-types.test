/* A C-style comment */
x: integer;
y: integer = 123; // A C++ style comment
b: boolean = false;
c: char    = 'q';
s: string  = "hello bminor";

print x, y, b, c, s;
