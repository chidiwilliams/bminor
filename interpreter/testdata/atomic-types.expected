0123false39hello bminor